examples/db-templates/mysql-template.json

```diff
@@ -40,6 +40,23 @@
       }
     },
     {
+      "kind": "PersistentVolumeClaim",
+      "apiVersion": "v1beta3",
+      "metadata": {
+        "name": "${PVC_NAME}"
+      },
+      "spec": {
+        "accessModes": [
+          "ReadWriteMany"
+        ],
+        "resources": {
+          "requests": {
+            "storage": "1Gi"
```

Comment:
if you specify wrong value, the List created from this template will fail to create, I don't think we need more validation